### DISCUSSION TITLE
Copying text inside a document field component block copies the whole component block

### DISCUSSION THREAD

**Discussion by raveling** [keystonejs/keystone#8615] [Questions] [answered]
### Discussed in https://github.com/keystonejs/keystone/discussions/7507

<div type='discussions-op-text'>

<sup>Originally posted by **PhilBookst** April 30, 2022</sup>
Hey Keystone Team,

When copying text inside a component block, keystone copies the whole component block structure instead of just the text.
This is really annoying when you start working on big texts and you cannot copy the text inside the blocks.

Is this the desired behaviour? If so, is there a way to opt-out of it? 

https://user-images.githubusercontent.com/68562883/166098469-ba8529aa-148b-4c40-a069-2747f463df6e.mp4

</div>

**Comment by ZowWeb** [comment, marked as answer]
@raveling if you're not concerned about the formatting of the text (bold, italics, color etc) you can paste it as plaintext with `Cmd + Shift + V`
